.github/scripts/end2end/run-e2e-ctst.sh

```diff
@@ -61,6 +61,11 @@ KAFKA_CLEANER_INTERVAL=$(kubectl get zenko ${ZENKO_NAME} -o jsonpath='{.spec.kaf
 # Setting CTST world params
 WORLD_PARAMETERS='{"subdomain":"'${SUBDOMAIN}'","ssl":false,"port":"'${ZENKO_PORT}'","AccountName":"'${ZENKO_ACCOUNT_NAME}'","AdminAccessKey":"'${ADMIN_ACCESS_KEY_ID}'","AdminSecretKey":"'${ADMIN_SECRET_ACCESS_KEY}'","VaultAuthHost":"'${VAULT_AUTH_HOST}'","NotificationDestination":"'${NOTIF_DEST_NAME}'","NotificationDestinationTopic":"'${NOTIF_DEST_TOPIC}'","NotificationDestinationAlt":"'${NOTIF_ALT_DEST_NAME}'","NotificationDestinationTopicAlt":"'${NOTIF_ALT_DEST_TOPIC}'","KafkaHosts":"'${KAFKA_HOST_PORT}'","KeycloakPassword":"'${KEYCLOAK_TEST_PASSWORD}'","KeycloakHost":"'${KEYCLOAK_TEST_HOST}'","KeycloakPort":"'${KEYCLOAK_TEST_PORT}'","keycloakRealm":"'${KEYCLOAK_TEST_REALM_NAME}'","keycloakClientId":"'${KEYCLOAK_TEST_CLIENT_ID}'","keycloakGrantType":"'${KEYCLOAK_TEST_GRANT_TYPE}'","StorageManagerUsername":"'${STORAGE_MANAGER_USER_NAME}'","StorageAccountOwnerUsername":"'${STORAGE_ACCOUNT_OWNER_USER_NAME}'","DataConsumerUsername":"'${DATA_CONSUMER_USER_NAME}'","ServiceUsersCredentials":'${SERVICE_USERS_CREDENTIALS}',"InstanceID":"'${INSTANCE_ID}'","KafkaCleanerInterval":"'${KAFKA_CLEANER_INTERVAL}'"}'
 
+# Set up environment variables for testing
+kubectl set env deployment end2end-connector-cloudserver SCUBA_HEALTHCHECK_FREQUENCY=100
+kubectl rollout status deployment end2end-connector-cloudserver
+kubectl patch cronjob end2end-ops-count-items -p '{"spec":{"suspend":true}}'
```

Comment:
Note for reviewers:

- Setting the healthcheck frequency to something fast will ensure the tests can be executed without issue; as scuba is disable till count items ran at least once (if no bucket, no account in the metrics, no quota checks)
- The count items might affect the existing tests, as it runs once per hour, but the tests are expecting "realtime" metrics (obtained today with the inflights).

Reply:
what is this "suspend" flag?
* will this be stable (vs adding locations...) ?
* if the cronjob is suspended, how will be get the initial metrics?

Can't we keep the cronjob (mostly useless, probably), and just run an extra job in parallel if needeD?

Reply:
Suspended avoids the cronjob from running, unless we trigger it from the tests

The risk of keeping it is flakiness if it runs exactly when we execute some quota tests... In this case, it can change the metrics and erase some inflights, and this might cause some tests to fail.
The risk is quite low, so we could choose to keep the cronjob, I will put it back and we'll disable it later if it causes issues